app/utils/polygon-forum-import-data/index.js

```diff
@@ -0,0 +1,155 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const getQuestionsAndUsers = async() => {
+    let users = [];
+    const questions = [];
+
+    for (let i = 1; i < 11; i++) {
+        const response = await (await fetch(`https://forum.matic.network/latest.json?page=${i}`)).json();
+
+        const newUsers = response.users.map(
+            ({id, username}) => ({id, name: username})
+        );
+
+        const newQuestions = response.topic_list.topics.map(
+            ({id, title, category_id, created_at}) => ({
+                id,
+                title,
+                category: category_id,
+                postTime: new Date(created_at).getTime()
+            })
+        );
+
+        newQuestions.forEach(question => questions.push(question));
+        newUsers.forEach(user => users.push(user));
+    }
+
+    users = users.filter((value, index, self) =>
+        index === self.findIndex((t) => (t.id === value.id))
+    );
+
+    return [questions, users];
+}
+
+const getAnswers = async(questions) => {
+    const questionIds = questions.map(question => question.id);
+    const urls = [];
+
+    for (const question of questionIds) {
+        const topic = await (await fetch(`https://forum.matic.network/t/${question}.json`)).json();
+        const newUrls = topic.post_stream.posts.map(({id}) => ({id}));
+        newUrls.forEach(url => urls.push(`https://forum.matic.network/posts/${url.id}.json`));
+    }
+
+    const answersList = [];
+
+    for (const url of urls) {
+        const post = await (await fetch(url)).json();
```

Comment:
please separate lines with await fetch and response.json() for better readability